circle.yml

```diff
@@ -16,6 +16,9 @@ dependencies:
 test:
   override:
     - reaction test
+    - npm install -g jest
+    - npm update
+    - jest
```

Comment:
Good comment but we probably don't want to call this just `test` as we already have 3 other test runners going on here. So maybe `test-react`? Or just call the script command "jest"?